src/Microsoft.AspNetCore.Mvc.Core/Internal/NonDisposableStream.cs

```diff
@@ -108,49 +108,6 @@ public override Task<int> ReadAsync(byte[] buffer, int offset, int count, Cancel
             return _innerStream.ReadAsync(buffer, offset, count, cancellationToken);
         }
 
-#if NET46
-        /// <inheritdoc />
-        public override IAsyncResult BeginRead(
```

Comment:
Aren't these methods part of the .NET Core surface? Need to be overridden if they're now included.